.github/workflows/tests.yml

```
@@ -4,9 +4,12 @@ on:
     branches:
       - main
       - master
-  pull_request:
+pull_request:
```

```suggestion
  pull_request:
```
Oops, GitHub suggestions mangled the spacing here for some reason.